src/resources/views/product/_form.blade.php

```diff
@@ -35,6 +35,25 @@
     </div>
 </div>
 
+<div class="form-row">
+    <div class="form-group col-12 col-md-6 col-xl-4">
+        <div class="input-group">
+            <span class="input-group-addon">
+                <i class="zmdi zmdi-code-setting"></i>
+            </span>
+            {{ Form::text('stock', null, [
```

Comment:
I think `number` type suits better to `stock` field. Also it's a better option for user experience since it is not accepting non-numeric input, supports incrementing/decrementing by buttons etc.

```suggestion
            {{ Form::number('stock', null, [
```

Reply:
Yes, number looks a better choice.

Reply:
@ChristopherKSmith Could you please accept the suggested change from @yusufkandemir?
(I don't have permission to do that)